### DISCUSSION TITLE
I've used this command to load the license, but I still encounter the following error when running 'finn.' Does anyone know the reason?

### DISCUSSION THREAD

**Discussion by harrisondusa** [Xilinx/finn#1062] [Q&A]
![螢幕擷取畫面 (15)](https://github.com/Xilinx/finn/assets/142143716/4fe7ad0c-c2b5-44c2-86a6-c6b7421a2e2f)
![螢幕擷取畫面 (22)](https://github.com/Xilinx/finn/assets/142143716/fc990591-d8c1-40ea-af17-35b7d3f5e399)
![螢幕擷取畫面 (23)](https://github.com/Xilinx/finn/assets/142143716/b073003c-28bd-40b1-86c5-e3da376fe7cf)

I've used this command to load the license, but I still encounter the following error when running 'finn.' Does anyone know the reason?
but License manager shows that Vivado ML Enterprise Edition has been successfully loaded.Is this a system bug?

**Comment by fpjentzsch** [comment]
This should work. Did you check inside the container (just running ./run_docker.sh) if the file is really accesible and that XILINXD_LICENSE_FILE is set?

**Reply by harrisondusa** [reply]
Hello, when I execute the command ./run_docker.sh, there is no response. So, I tried executing ./run-docker.sh instead. It displayed the following screen. Would it show that a license has been detected here if everything is normal?
![螢幕擷取畫面 (27)](https://github.com/Xilinx/finn/assets/142143716/8bc6fad4-4756-4e2c-be4f-6132e70a6a2d)

**Reply by harrisondusa** [reply]
![螢幕擷取畫面 (28)](https://github.com/Xilinx/finn/assets/142143716/c5b178ed-a32c-41f3-8519-d3cc3c4eb7cb)
It detected this path.
